client.go

```diff
@@ -40,17 +59,20 @@ type client struct {
 	restClient rest.Client
 }
 
-// NewClientForConfig creates a new Site24x7 API Client for Config c.
-func NewClientForConfig(c Config) Client {
-	oauthConfig := oauth.NewConfig(c.ClientID, c.ClientSecret, c.RefreshToken)
-
-	httpClient := oauthConfig.Client(context.Background())
+// New creates a new Site24x7 API Client with Config c.
+func New(c Config) Client {
```

Comment:
Function name maybe to general. p.e. ClientWithConfig

Reply:
It is generally good practice to use `New` as the constructor for the main type in a package.